packages/cos-features/features/harvester.yaml

```diff
@@ -0,0 +1,43 @@
+stages:
+   network:
+     - name: "Setup k3s"
+       directories:
+       - path: "/usr/local/bin"
+         permissions: 0755
+         owner: 0
+         group: 0
+       - path: "/run/harvester"
```

Comment:
Shouldn't it be part of the harvester step? I think it is good that `Setup k3s` is equivalent for all. So in the future if we want to add some sort of yaml dependency structure to pull _stages_ from another one it will be straight forward to adapt these files :)

Reply:
Good catch! fixed in https://github.com/rancher-sandbox/cOS-toolkit/pull/98/commits/37d5e07984bf4aa995ef28260d8f55f9e5dcaa86 :+1: